packages/send/src/sendFile.ts

```diff
@@ -49,9 +49,11 @@ export const sendFile = <Request extends I = I, Response extends S = S>(_: Reque
 
   const stream = createReadStream(root ? root + path : path, options)
 
-  stream.on('error', (err) => void cb(err))
+  if (cb !== undefined) {
```

Comment:
if cb is null it can also error. Better do non-strict checking like `cb != null` so it won't execute for neither `undefined`, nor `null` nor `false`